codec/packer.go

```diff
@@ -139,6 +140,32 @@ func (p *Packer) UnpackInt(required bool) int {
 	return int(v)
 }
 
```

Comment:
We will also need some utility functions to determine the size of encoding an int/uint to properly determine bandwidth usage of transaction.

Reply:
In this PR ?